index.md

```diff
@@ -176,8 +183,8 @@ work in archival journals or conferences.
 
 **Important Dates:**
 * Submission deadline: April 5th
-* Notification to authors: ~~April 19th~~ April 24th
-* Camera ready deadline: ~~April 26th~~ May 1st
+* Notification to authors: April 19th
+* Camera ready deadline: April 26th
```

Comment:
was this intentional? We could change to
```suggestion
* Notification to authors: April 24th
* Camera ready deadline: May 1st
```

Reply:
Ah, wasn't intentional, I messed up the branching commit. 
Fixed it now.